app/src/main/java/fr/free/nrw/commons/LocationPicker/LocationPickerActivity.java

```diff
@@ -113,7 +124,7 @@ public class LocationPickerActivity extends BaseActivity implements OnMapReadyCa
      */
     private ImageView shadow;
     /**
-     * largeToolbarText : textView of shadow
+     * cGPS : textView of centre to location
```

Comment:
This should be 
/**
     * largeToolbarText : textView of shadow
     */
I forgot to change it back.